go.mod

```diff
@@ -1,6 +1,7 @@
 module k8s.io/ingress-nginx
 
-go 1.21
+go 1.22.0
```

Comment:
can you do go 1.22.3?

Reply:
I could, but that's also not the case on main. Since my PR kind of bloats the `go.work.sum`, I was thinking about just copying the `go.*` files from `main` to `release-1.10`. The diff doesn't look that bad...